### DISCUSSION TITLE
Release Partners 9/2/2021

### DISCUSSION THREAD

**Discussion by joshblack** [carbon-design-system/carbon#9589] [Release Feedback]
This week is a lead-up to our Beta 2 releases. We want to quickly go over the feedback we received from Beta 1 that we're incorporating into Beta 2. We also want to cover the latest news for Beta 2 and start talking about education, migration, and tooling.

**Agenda**

- Feedback from Beta 1
- Looking towards Beta 2
- Education topics and events, in particular with PAL teams
- Initial round table for coordinating migration, migration process feedback and tooling

## Feedback from Beta 1

- Docs updates
- Addressed missing or incorrect exports
- Automatically install peer dependencies
- More docs and blog posts on topics that are changing

## Looking towards Beta 2

We were hoping to have a release of Beta 2 on September 2nd. Unfortunately, we weren't able to deliver on the items below. These will be worked on in the next sprint and our next beta data will be September 16th. 

- Component prop changes
- Updates to components
  - Tooltip
  - ContentSwitcher
  - Notification
  - Dialog

## Education events

We would like to start preparing a series of content to talk about changes in v11 and how to migrate and use these new topics. This content should be presented both in a regular series along with tying into sessions that local groups do across IBM. In particular, working with existing design/development/PAL events would be preferred.

Some initial topics included:

- An overview of CSS Grid and what it means for design and development
- Understanding the changes to design tokens, could contain a section on light/dark mode to emulate what Morgana D'Almeida did recently
- How to plan your migration to v11 (design, development)
- Moving to Sass Modules

## Migration 

We would like to have a solid template or overview for how teams can plan the migration of their product/platform/org to v11. This was something that noticeably was missing from v10 and teams had to come up with these plans themselves. Obviously, there will be some balance of what we can provide and how useful it is, but it'd be great to get this conversation started to see what teams would like to have as guidance in this space.

Some tooling or resources that we are planning to provide so far:

- Migration docs for design and development
- Migration tooling
- Educational series

**Comment by adrianamorales** [comment]
**Additiona education event:**
- Inline theming / properly using Carbon themes — in Cloud PAL we got a lot of questions from teams on how to do mix themes. Teams frequently use the wrong tokens or aren't aware of how to set one section as light and the other section as dark. 
- Layer tokens - this is a new concept for designers and developers. The v11 docs do a great job explaining it, but I know from past PAL experience teams want the TL;DR synopsis of concepts.

**Reply by aagonzales** [reply]
@adrianamorales I would love to get your feedback on [some updates](https://carbondesignsystem-git-fork-aagonza-1d14f8-carbon-design-system.vercel.app/guidelines/color/implementation#implementing-layering) I'm working on for the layer tokens docs. I've added in some "when to use" sections that I think could help as a TL;DR. I'd also love to pick your brain about what other TL;DR experiences might be like.

**Reply by adrianamorales** [reply]
@aagonzales of course! I'm available this week and next!
